CHANGES.rst

```diff
@@ -1,9 +1,19 @@
 Changelog for zest.releaser
 ===========================
 
-5.8 (unreleased)
+6.0 (unreleased)
 ----------------
 
+- Made ``twine`` a core dependency.  We now always use it for
+  registering and uploading.  We require at least version 1.6.0.
+  [maurits]
+
+- When uploading with ``twine`` first use the ``twine register``
+  command.  This needs ``twine`` version 1.6.0 or higher, which you
+  get with the ``zest.releaser[recommended]`` extra.
```

Comment:
If it is a core dependency, you don't need to install it via "recommended" anymore :-)